packages/odyssey-storybook/src/components/odyssey-mui/Infobox/Infobox.mdx

```diff
@@ -116,6 +116,97 @@ Infobox should always include their main messaging within the body content. Titl
 
 When deploying Infoboxes, two `role`s may apply. Error variants should utilize the `alert` role, as their contents represent immediate risk or failure. Success and Warning variants are better suited to the `status` role, which provides a less urgent announcement on appearance. The Info variant does not require a `role`, as it represents plain content.
 
-## Props
-
-<ArgsTable />
+## References
+
+<figure class="ods-table--figure">
+  <figcaption class="ods-table--figcaption">Related Components</figcaption>
+  <table class="ods-table">
+    <caption>Differences between the available alert components</caption>
+    <thead>
+      <tr>
+        <th scope="column">Component</th>
+        <th scope="column">Available Variants</th>
+        <th scope="column">
+          <span class="has-ods-tooltip">
+            <abbr aria-describedby="tip-dynamic">Dynamic</abbr>
+            <aside
+              class="ods-tooltip is-ods-tooltip-top"
+              id="tip-dynamic"
+              role="tooltip"
+            >
```

Comment:
Why these classes?

Reply:
These are a holdover from the legacy docs. I copy/pasted them here for consistency.

Reply:
Could we get rid of them?